src/validations/Dockerfile

```diff
@@ -1,8 +1,3 @@
 FROM openjdk:17-alpine
 
-# Install htmltidy
-RUN apk add --no-cache \
-    bash \
-    tidyhtml
-
 WORKDIR /root
```

Comment:
The Dockerfile isn't necessary anymore - I pushed up a commit that removes it in favor of using a Maven image.